group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinatorShard.java

```diff
@@ -306,6 +308,22 @@ public CoordinatorResult<OffsetCommitResponseData, Record> commitOffset(
         return offsetMetadataManager.commitOffset(context, request);
     }
 
+    /**
+     * Handles a LeaveGroup request.
+     *
+     * @param context The request context.
```

Comment:
The formatting with tab spaces between context and the description seems different in this javadoc and the next one, do we wanna uniformly space them?

Reply:
will leave it as is; it's the same formatting like the other old apis. There is no tab space in between the parameter name and the description